tests/ui/llm_studio.feature

```diff
@@ -0,0 +1,30 @@
+Feature: LLM Studio
+
+    Background: LLM Studio user
```

Comment:
That test would need to be excluded from non-cloud ui tests.

Reply:
All ui tests are cloud only ui tests @maxjeblick. This PR only addresses UI testing on Cloud